### PR TITLE
Translation from Carmen-Grijalba/TEI

### PR DESCRIPTION
**Pull request by Carmen-Grijalba**

### CODE REVIEW

**Comment by sabineseifert** [comment]
Merged dev branch into pull request to resolve conflicts. Ready to review for @HelenaSabel. :-)

File: P5/Source/Specs/unclear.xml [hunk 1]
```diff
@@ -69,20 +68,24 @@
...
           <gloss versionDate="2018-01-17" xml:lang="de">ausgebleicht</gloss>
         </valItem>
         <valItem ident="background_noise">
           <gloss xml:lang="en" versionDate="2018-01-17">background_noise</gloss>
+          <gloss xml:lang="es" versionDate="2022-02-20">ruido de fondo</gloss>
           <gloss versionDate="2018-01-17" xml:lang="de">Hintergrundlärm</gloss>
         </valItem>
         <valItem ident="eccentric_ductus">
           <gloss xml:lang="en" versionDate="2018-01-17">eccentric_ductus</gloss>
+          <gloss xml:lang="es" versionDate="2022-02-20">conducto excéntrico</gloss>
```
Comment (HelenaSabel) [inline P5/Source/Specs/unclear.xml]:
```suggestion
          <gloss xml:lang="es" versionDate="2022-06-03">ductus excéntrico</gloss>
```

File: P5/Source/Specs/unclear.xml [hunk 2]
```diff
@@ -185,6 +187,9 @@ his att<unclear>ention</unclear>
 concerned, as further discussed in section <ptr target="#PHDH"/>.</p>
 
   </remarks>
+  <remarks versionDate="2022-02-20" xml:lang="es"><p>El mismo elemento se usar para todos los casos de incertidubre en la transcripción del contenido de un elemento, para material tanto escrito como hablado. Para otros aspectos de certeza, incertidumbre y fiablilidad del etiquetado y transcripción, consultar el capítulo <ptr target="#CE"/>.</p>
```
Comment (HelenaSabel) [inline P5/Source/Specs/unclear.xml]:
```suggestion
  <remarks versionDate="2022-02-20" xml:lang="es"><p>El mismo elemento se usa para todos los casos de incertidubre en la transcripción del contenido de un elemento, para material tanto escrito como hablado. Para otros aspectos relacionados con la certeza, incertidumbre y fiablilidad del etiquetado y transcripción, consultar el capítulo <ptr target="#CE"/>.</p>
```

File: P5/Source/Specs/unclear.xml [hunk 3]
```diff
@@ -185,6 +187,9 @@ his att<unclear>ention</unclear>
...
+    <p>Los elementos <gi>damage</gi>, <gi>gap</gi>, <gi>del</gi>, <gi>unclear</gi> y <gi>supplied</gi> pueden estar estrechamente relacionados en su uso. Consulta la sección <ptr target="#PHCOMB"/> para ver una discusión sobre qué elemento es apropiado en qué circunstancias.</p>
+    <p>El atributo <att>hand</att> dirige a una definición de la mano impliciada, más discutido en la sección <ptr target="#PHDH"/>.</p></remarks>
```
Comment (HelenaSabel) [inline P5/Source/Specs/unclear.xml]:
```suggestion
    <p>El atributo <att>hand</att> dirige a una descripción de la mano implicada; se discute de manera más detallada en la sección <ptr target="#PHDH"/>.</p></remarks>
```

**Review by HelenaSabel — CHANGES_REQUESTED** [review]
I left some suggestions that should be commited before merging (some are stylistic, but others concern translation issues and typos)

**Pull request merged**